test/e2e/conformance/validatingadmissionpolicy_test.go

```diff
@@ -291,6 +292,17 @@ func TestValidatingAdmissionPolicyCrossWorkspaceAPIBinding(t *testing.T) {
 	_, err = kcpClusterClient.Cluster(sourcePath).ApisV1alpha2().APIExports().Create(ctx, cowboysAPIExport, metav1.CreateOptions{})
 	require.NoError(t, err)
 
+	t.Logf("Creating default namespace in target workspace")
```

Comment:
yap, this is not the fix. + it will try to create namespace where workspace bootstrap process does the same. THis might cause other isues